src/editors/data/redux/thunkActions/requests.test.js

```diff
@@ -78,12 +89,13 @@ describe('requests thunkActions module', () => {
   const testNetworkRequestAction = ({
     action,
     args,
+    testState,
     expectedData,
     expectedString,
   }) => {
     let dispatchedAction;
     beforeEach(() => {
-      action({ ...args, onSuccess, onFailure })(dispatch);
+      action({ ...args, onSuccess, onFailure })(dispatch, testState);
```

Comment:
```suggestion
      action({ ...args, onSuccess, onFailure })(dispatch, () => testState);
```
and should use the top-level `testState` object, not one passed in here